<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2207</width>
    <height>905</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(25, 35, 45); color: rgb(255,255,255);</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Pre-processing</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QGroupBox" name="Smooth_Group">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="toolTipDuration">
           <number>-1</number>
          </property>
          <property name="statusTip">
           <string>applying scipy savgol_filter</string>
          </property>
          <property name="accessibleDescription">
           <string/>
          </property>
          <property name="title">
           <string>Peak finding intital thresholds</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="1">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>gradient peak height</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>absolute peak height</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="peak_init_th">
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string>20</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="grad_edge_thresh">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>2. step filters</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="1">
            <widget class="QLineEdit" name="grad_edge_ratio">
             <property name="text">
              <string>0.8</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>combi score filter</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>additional peak ratio</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLineEdit" name="combi_score_filter">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Decay determination</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>decay ratio of peak max</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLineEdit" name="decay_th">
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string>0.5</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="button_refresh">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(11, 76, 80)</string>
            </property>
            <property name="text">
             <string>refresh preview</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="peak_error_label">
            <property name="text">
             <string/>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDialogButtonBox" name="dialog_buttons">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(11, 76, 80)</string>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="standardButtons">
             <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
            </property>
            <property name="centerButtons">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="normalize_bool">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Normalize</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,3,1">
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Peak Data</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QTextEdit" name="peakDetails">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="gradpeakDetails">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Sample Data</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="MplWidget" name="MplWidget_peaks1" native="true">
               <property name="styleSheet">
                <string notr="true">margin: 5px;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="MplWidget" name="MplWidget_peaks2" native="true">
               <property name="styleSheet">
                <string notr="true">margin: 5px;</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="MplWidget" name="MplWidget_peaks3" native="true">
               <property name="styleSheet">
                <string notr="true">margin: 5px;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
             <property name="spacing">
              <number>60</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetFixedSize</enum>
             </property>
             <property name="leftMargin">
              <number>66</number>
             </property>
             <property name="rightMargin">
              <number>66</number>
             </property>
             <item>
              <widget class="QScrollBar" name="prepro_scroll">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="maximum">
                <number>1</number>
               </property>
               <property name="pageStep">
                <number>1</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="invertedAppearance">
                <bool>false</bool>
               </property>
               <property name="invertedControls">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="prepro_roll">
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Roll</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Statistics</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,1,1,1,0">
           <item>
            <widget class="QLabel" name="mousePosition_label">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="MplWidget" name="MplWidget_statistics_2" native="true">
             <property name="font">
              <font>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">padding: 5px;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="MplWidget" name="MplWidget_statistics" native="true">
             <property name="font">
              <font>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">padding: 5px;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="MplWidget" name="MplWidget_statistics_3" native="true">
             <property name="font">
              <font>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">padding: 5px;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_calculate_Combi">
             <property name="text">
              <string>calculate</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>dialog_buttons</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialog_buttons</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
